src/App.jsx

```diff
@@ -10,7 +10,7 @@ function App() {
     <>
       <Nav />
```

Comment:
You should have a way for the user to return home without using the browser's back button. Curently there is no working `Home` link